setup.py

```diff
@@ -47,6 +22,55 @@
     'extra_link_args': ['-fopenmp', '-g']
     }
 
+# Set compiler options
+# Windows
+if sys.platform == 'win32':
+    raise DistUtilsError('Windows is not supported.')
+# Mac OS X - needs gcc (usually via HomeBrew) because the default compiler LLVM (clang) does not support OpenMP
+#          - with gcc -fopenmp option implies -pthread
+elif sys.platform == 'darwin':
+    try:
+        sp.check_call('scripts/osx.sh', shell=True)
+    except sp.CalledProcessError:
+        raise DistutilsError('Failed to prepare Mac OS X properly. See earlier errors.')
+    gccpath = glob.glob('/usr/local/bin/gcc-[4-9]*')
```

Comment:
There was another GCC major release last week, so this line will break in the future.

Reply:
Fixed now with more careful gcc search.